buku.py

```diff
@@ -2565,17 +2565,28 @@ def get_firefox_profile_name(path):
     profile : str
         Firefox profile name.
     """
-
-    try:
-        for name in os.listdir(path):
-            # can be in the format nnnnnn.default or nnnnnn.default-nnnnnn...
-            if '.default' in name and os.path.isdir(os.path.join(path, name)):
-                logdbg(name)
-                return name
-    except FileNotFoundError:
-        pass
-
-    return None
+    from configparser import ConfigParser, NoOptionError
```

Comment:
For Windows it will not work. The profiles.ini is above the Profiles directory. Please ensure you have tested on Debian, Ubuntu, Windows and OS X. If you don't have access to any of these OS -es let me know.

Reply:
I'm gonna check Debian, Ubuntu and OS X. Please check Windows..

Reply:
Works on Ubuntu.

Reply:
OK. I'll test it on Windows.

Reply:
Debian also tested.